TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/BobbyPath.java

```diff
@@ -0,0 +1,32 @@
+package org.firstinspires.ftc.teamcode.drive.opmode;
+
+import com.acmerobotics.roadrunner.geometry.Pose2d;
+import com.acmerobotics.roadrunner.geometry.Vector2d;
+import com.acmerobotics.roadrunner.trajectory.Trajectory;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+
+import org.firstinspires.ftc.teamcode.drive.SampleMecanumDrive;
+@Autonomous (group = "drive")
+public class BobbyPath extends LinearOpMode {
+    @Override
+    public void runOpMode() {
+        SampleMecanumDrive drive = new Sampl eMecanumDrive(hardwareMap);
+
+        drive.setPoseEstimate(new Pose2d(50, 50, Math.toRadians(270)));
+        Trajectory cool = drive.trajectoryBuilder(new Pose2d(50, 50, 270))
+                        .lineTo(new Vector2d(50, -50)).build()
```

Comment:
'.build()' is missing a semicolon

Reply:
yo build aint even in the right location😭